README.md

```diff
@@ -58,7 +58,22 @@ flake8 --select=DOC <FILE_OR_FOLDER>
 If you don't include `--select=DOC` in your command, _flake8_ will also run
 other built-in _flake8_ linters on your code.
 
-### 2.3. Native vs _flake8_
+### 2.3. As a pre-commit hook
+
+`pydoclint` is configured for [pre-commit](https://pre-commit.com/)
+and can be set up as a hook with the following `.pre-commit-config.yaml`
+configuration:
+
+```yaml
+- repo: https://github.com/jsh9/pydoclint
+  rev: 0.0.7
+  hooks:
+    - id: pydoclint
```

Comment:
Could you show an example or two on how to specify options in the pre-commit hook?

Reply:
Added an example of passing the config via args.